testsuite/openmodelica/fmi/CoSimulation/2.0/BooleanNetwork1.mo

```diff
@@ -0,0 +1,89 @@
+model BooleanNetwork1 "Demonstrates the usage of blocks from Modelica.Blocks.MathBoolean"
```

Comment:
How i this model different from `Modelica.Blocks.Examples.BooleanNetwork1`?

Reply:
Different style of model name, but removed for now.